### PR TITLE
Replace abstract actions/stores with their decorator analogs

### PR DESCRIPTION
**Pull request by maratfakhreev**
This PR created refer to #13 

This PR replace all abstract entities with their mixin (implement via ES7 decorators) analogs. Thus, we follow the "Composition over inheritance" principle.

### CODE REVIEW

**Comment by AmirMingaliev** [comment]
🚬 🐤

File: app/mixins/stores.jsx [hunk 1]
```diff
@@ -0,0 +1,13 @@
+export const ModalStoreMixin = {
```
Comment (EmilShakirov) [inline app/mixins/stores.jsx]: Is it ok that we have difference between mixin name and file name?

**Comment by VladimirMikhailov** [comment]
looks good, but what if could put the code from 'mixins/actions' and `mixins/stores' into modal subdirectory or rename it in order to make it a bit cleaner for further usage?

**Comment by maratfakhreev** [comment]
@VladimirMikhailov @EmilShakirov Thank you for the comments, they were applied

**Pull request merged**